backend/src/core/annotation.py

```diff
@@ -98,46 +94,53 @@ def process_tasks(annotation_queue, genomic_unit_collection):  # pylint: disable
-                            - Canceling Annotation, Missing {missing_dependencies} ...\n"
+                        logger.info(
+                            '%s Canceling Annotation, Missing %s ...',
+                            annotation_unit_string(genomic_unit['unit'], dataset_json['data_set']), missing_dependencies
                         )
+
                     continue
 
                 task = AnnotationTaskFactory.create(genomic_unit, dataset_json)
-                log_to_file(f"{genomic_unit['unit']} for {dataset_json['data_set']} - Creating Task To Annotate...\n")
-
-                annotation_task_futures[executor.submit(task.annotate)] = (
-                    genomic_unit,
-                    task,
+                logger.info(
+                    '%s Creating Task To Annotate...',
+                    annotation_unit_string(genomic_unit['unit'], dataset_json['data_set'])
                 )
 
+                annotation_task_futures[executor.submit(task.annotate)] = (genomic_unit, task)
+
                 for future in concurrent.futures.as_completed(annotation_task_futures):
                     genomic_unit, annotation_task = annotation_task_futures[future]
-                    log_to_file(f"{genomic_unit['unit']} for {dataset_json['data_set']} - Query completed...\n")
+                    logger.info(
+                        '%s Query completed...', annotation_unit_string(genomic_unit['unit'], dataset_json['data_set'])
+                    )
+
                     try:
                         result_temp = future.result()
 
                         for annotation in annotation_task.extract(result_temp):
-                            log_to_file(
-                                f"{genomic_unit['unit']} for {annotation_task.dataset['data_set']} - \
-                                Saving {annotation['value']}...\n"
+                            logger.info(
+                                '%s Saving %s...',
+                                annotation_unit_string(genomic_unit['unit'], annotation_task.dataset['data_set']),
+                                annotation['value']
                             )
                             genomic_unit_collection.annotate_genomic_unit(annotation_task.genomic_unit, annotation)
 
                     except FileNotFoundError as error:
-                        log_to_file(f"exception happened {error} with {genomic_unit} and {annotation_task}\n")
+                        logger.info(
+                            '%s exception happened %s with %s and %s', annotation_log_label(), error, genomic_unit,
+                            annotation_task
+                        )
 
-                    log_to_file("\n")
                     del annotation_task_futures[future]
 
-            log_to_file("after for loop for waiting for all of the futures to finish\n\n")
+            logger.info("%s ...after for loop for waiting for all of the futures to finish", annotation_log_label())
```

Comment:
Maybe we can update this log to match closer to what the initiating log for this background task says "Processing annotation tasks queue" from https://github.com/uab-cgds-worthey/rosalution/pull/108/files#diff-9f447bc75c68f58ec057bb592e1066030b1de98a6d7c5b961cdf7b4bed582af7R64 to be like  "Processing annotation tasks queue complete", you can probably come up with something that makes a little bit more sense then my suggestion

Reply:
I just changed it to be "Processing annotation tasks queue complete".